lib/psi.rb

```diff
@@ -1092,24 +1071,32 @@ def processTBD(os_model, io_f, schema_f, psi_set)
         next unless io_p.set.has_key?(p)
 
         edges.values.each do |edge|
-          next unless edge.has_key?(:psi)
+          next unless edge.has_key?(:psi)   # open to transition edges TO DO ...
```

Comment:
By default, _mild_ transitions between 2 surfaces are ignored in TBD. In most cases such edges are either artefacts of the model (e.g. from tessallation) or subtle differences in construction (e.g. change in cladding, or an extra inch of insulation), and as such should not be considered (by default) as _major_ thermal bridges. There will be cases where this presumption is flat out wrong (e.g. structural transitional joints, curtain wall spandrel perimeters along other wall types). Adding another PSI type is likely the best way moving forward.